app/src/main/java/co/nimblehq/compose/crypto/ui/screens/detail/DetailScreen.kt

```diff
@@ -214,6 +213,20 @@ private fun DetailScreenContent(
                     sellBuyLayoutHeight = sellBuyLayoutHeight.value,
                     coinDetailUiModel = coinDetailUiModel
                 )
+
+                if (showLoading) {
+                    CircularProgressIndicator(
+                        modifier = Modifier
+                            .constrainAs(progressIndicator) {
+                                linkTo(
+                                    start = parent.start,
+                                    end = parent.end,
+                                    top = parent.top,
+                                    bottom = parent.bottom
+                                )
+                            },
+                    )
+                }
             }
 
             if (showLoading) {
```

Comment:
Why do we need 2 `CircularProgressIndicator`s?

Reply:
@luongvo Ops. Rebase issue